javascripts/discourse/components/spreadsheet-editor.hbs

```diff
@@ -46,8 +46,12 @@
           {{/if}}
         </div>
       {{/if}}
-      <DPopover>
-        <DButton class="trigger" @icon="question" />
+      <DTooltip
+        @icon="question"
+        @triggers="click"
```

Comment:
Is `@triggers` DTooltip's api?

Reply:
yes

Reply:
by default its ["hover", "click"]

Reply:
Oh I did `makeArray(this.triggers)` that's why

Reply:
Yes, and I added the trigger to be click only so it maintains the same behaviour it had with `<DPopover/>` originally